DEPLOY_NEW_HOWTO.md

```diff
@@ -0,0 +1,198 @@
+# Building a new Django application
+
+This guide will walk you through the steps need to create and deploy a new
+Django application. It will then show you how you can modify the application
+and then redeploy it.
+
+## Before you begin
+
+Before running this guide, you must install the Cloud SDK, Cloud SQK Proxy
+and setup your development environment.
+
+1. Download and install the
+   [Cloud SDK](https://cloud.google.com/sdk/docs/quickstarts)
+
+2. Download and install the
+   [Cloud SQL Proxy](https://cloud.google.com/sql/docs/mysql/connect-admin-proxy#install)
+
+3. Download and install [Docker](https://docs.docker.com/install/overview/)
+   (any edition)
+
+4. Prepare your environment for Python development. You have the Python 3.5 or
+   later, pip and virtualenv installed on your system. For instructions, refer
+   to the [Python Development Environment Setup Guide](https://cloud.google.com/python/setup).
+
+5. If you don't already one, you need to create a 
+   [Google Cloud Platform billing account](https://console.cloud.google.com/billing).
+   If you are new to Google Cloud Platform, you may be able to take advantage of
+   a [free trial](https://cloud.google.com/free/).
+
+## Types of deployment
+
+When deploying your Django application, you can choose which Google serving
```

Comment:
Is there another way to phrase Google serving tecnology?

Reply:
Google hosting technology? Google deployment technology?